src/emitter.cpp

```diff
@@ -112,7 +111,7 @@ namespace Sass {
     wbuf.buffer = text + wbuf.buffer;
   }
 
-  char Emitter::last_char()
+  const char& Emitter::last_char()
```

Comment:
nit: `char` is tiny, it's better to return by value.

Reply:
Done